### ISSUE TITLE
Proposal: use low power lasers instead of led

### ISSUE THREAD

**Issue by allegfede** [open-laser-tag/tagger#12]
What do you think about using a cheap low power laser diode (maybe an IR one) to replace the led's one?
Should range be increased and the "sharness"  also, resulting in a more "real" little hitting spot.

**Comment by gldhnchn** [comment]
Yes, I already thought about this. Maybe it is a good idea to increase the shooting distance and to reduce the size of the hitting spot. On the other hand, if the size of the hitting spot is too small, it could be frustrating, because it will be too difficult to hit. After all this idea is something that just has to be tested.
This project was kind of on hold for the last 2 years. Sorry for the late response. The reason is, that we haven't made it to a reliable playable hardware situation and so the motivation dropped at some point. The infrared stuff was exactly the main problem of our development. It didn't work reliable for distances above a few meters. So maybe maybe the idea of switching to a laser diode could be the solution of our old problems.

**Comment by 46cv8** [comment]
I would highly suggest if anyone is going to switch to low power laser diodes they have some way to confirm there diodes are infact low power and that the wave lengths being emitted are accurate. I suppose if you sourced the laser modules from digikey or mouser it wouldn't be an issue, but I'de be concerned about relatively high power IR laser diodes being sold as low power ones on aliexpress and the like. I've seen this reported by at least one youtuber in the past. It was something like 50x more powerful than the claimed power, and it also had a bad dichroic filter in it that was pretty leaky (the dichroic isn't as important for IR lasers as I understand it, it's more important when you are using frequency doubling to block IR or RED and only let through green/blue.).
